config/core/test-configs.yaml

```diff
@@ -1910,6 +1911,7 @@ test_configs:
       - kselftest-lib
       - kselftest-livepatch
       - kselftest-lkdtm
+      - kselftest-rtc
```

Comment:
OK with one failure: https://staging.kernelci.org/test/plan/id/61e84d3088d83e941f2178dd/

and 2 failures with Clang:
https://staging.kernelci.org/test/plan/id/61e849bacf88d41e5b217807/

Reply:
Those are the same failure. Must be a timing issue whether LAVA decides to report both (the test case it was running as well as the test suite that contains it) or just one as failed.